src/pudl/transform/eia860.py

```diff
@@ -491,10 +483,18 @@ def boiler_generator_assn(eia860_dfs, eia860_transformed_dfs):
     b_g_df["boiler_id"] = b_g_df["boiler_id"].astype(str)
 
     # This drop_duplicates isn't removing all duplicates
```

Comment:
Any idea what's up with this comment?

Reply:
I... I assume this lil bb was added before the chunk above it, which says 
```
# We need to cast the generator_id column as type str because 
# it is heterogeneous int/str which make drop_duplicates fail.
```

So I'd like to remove it

Reply:
haha this was me 5 years ago in the same commit so yeah i'm removing it

![image](https://user-images.githubusercontent.com/25487998/184973989-b660be2f-4a23-458d-ba97-06275ea18e7b.png)